koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -476,7 +490,7 @@ def handler(self, src, target_info, arch, scratch=False,
             'osbs_build_id': build_id,
             'files': [],
             'repositories': repositories,
-            'koji_build_id': koji_build_id,
+            'koji_build_id': koji_build_id
```

Comment:
Generally it's nice to leave a terminating "," in a list or dict, so that adding another line at the end doesn't trip up by forgetting to do it.